### PR TITLE
Added homework8

### PR DESCRIPTION
**Pull request by polina-pavlova**

### CODE REVIEW

File: homework8/tasks/task1.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from keyword import iskeyword
+
+
+class KeyValueStorage:
```
Comment (PeppaTheC) [inline homework8/tasks/task1.py:4]: nice

File: homework8/tasks/task2.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    def __contains__(self, item):
+        cursor = self.connect_db()
+        cursor.execute(
+            f"SELECT * from {self.table_name} WHERE name=:item", {"item": item}
+        )
+        return cursor.fetchone()
+
+    def __getitem__(self, item):
+        cursor = self.connect_db()
```
Comment (PeppaTheC) [inline homework8/tasks/task2.py]: it woud be nice always have one conneciton to db for class

File: homework8/tasks/task2.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        cursor.execute(
+            f"SELECT * from {self.table_name} WHERE name=:item", {"item": item}
+        )
+        return cursor.fetchone()
+
+    def __iter__(self):
+        cursor = self.connect_db()
+        cursor.execute(f"SELECT * from {self.table_name}")
+        return cursor
```
Comment (PeppaTheC) [inline homework8/tasks/task2.py:36]: does it support 
for president in presidents:
           print(president['name'])

**Review by PeppaTheC — CHANGES_REQUESTED** [review]

File: homework8/tasks/task2.py (hunk 3 above)
Comment (PeppaTheC) [inline homework8/tasks/task2.py]: for president in presidents:
           print(president['name'])

Comment (polina-pavlova) [reply homework8/tasks/task2.py]: исправила

**Review by PeppaTheC — APPROVED** [review]

**Pull request merged**